packages/sdk/schema/src/types/field.ts

```diff
@@ -61,6 +62,29 @@ export const toFieldValueType = (type: AST.AST): FieldValueType => {
   return FieldValueType.JSON;
 };
 
+// TODO(ZaymonFC): Find all the appropriate annotations.
```

Comment:
let's establish how we're gonna store the schema.

For the discussion with @richburdon this morning, we want to remove `FieldType` in favour of storing everything in the [json-schema](https://json-schema.org/learn/getting-started-step-by-step) format.

There's an import disticution between the type schema and view schema:
Type schema defines the schema of the data stored in the database -- this is what the current `DynamicSchema`/`StoredSchema` is.
View schema defines the field config of the view which is backing the table.
Reusing the same storage format for both of those cases should make the code concceptually simpler.

For example: